validphys2/src/validphys/closuretest/multiclosure_output.py

```diff
@@ -10,11 +10,30 @@
 import pandas as pd
 import scipy.special as special
 import scipy.stats
+import yaml
 
 from reportengine.figure import figure, figuregen
 from reportengine.table import table
 
+
+from validphys.closuretest.multiclosure import expected_dataset_bias_variance
+from validphys.loader import Loader
+from validphys.closuretest.multiclosure import (
+    internal_multiclosure_data_loader,
+    fits_data_bias_variance,
+)
+from validphys.core import DataGroupSpec
+from validphys.covmats import dataset_inputs_covmat_from_systematics
 from validphys import plotutils
+import logging
+
+log = logging.getLogger(__name__)
+
+l = Loader()
```

Comment:
The loader should not be used in various modules. Use the existing mechanisms, e.g. in config .py to get data.